backend/impl/src/main/java/com/virtuslab/gitmachete/backend/impl/GitMacheteRepositorySnapshot.java

```diff
@@ -36,14 +36,14 @@ public class GitMacheteRepositorySnapshot implements IGitMacheteRepositorySnapsh
   @Getter
   private final Set<String> skippedBranchNames;
 
-  @Getter
-  private final Option<String> rebasedBranchName;
-
   private final PreRebaseHookExecutor preRebaseHookExecutor;
 
   @Getter
   private final OngoingRepositoryOperation ongoingRepositoryOperation;
```

Comment:
Actually... would it make sense to group the enum defining the operation (currently: `OngoingRepositoryOperation`) and the affected branch name into a single data class with 2 fields? 🤔

Reply:
Esp. that AFAICS the operation type and branch name are almost always derived&used together

Reply:
Sure, why not. Also will be more verbose than fiveWordLongNames